### PR TITLE
Fy24 q4

### PR DESCRIPTION
**Pull request by elmeraa-usgs**
# FY 2024 Quarter 3: River Conditions Animation
## Summary of changes:
- gathered all callouts from river conditions teams chat 
- Built twitter version 
## To do:
- [x] share with Joe and Cee for approval to post 
- [x] write up accompanying video description
- [ ] get drupal link and update twitter social media draft

### CODE REVIEW

File: callouts_cfg.yml [hunk 1]
```diff
@@ -174,7 +234,37 @@
...
+    y_loc: 0.185
+    # Numeric indicating where to place text relative to location: 1,2,3,4 (below, left, above, right)
+    pos: 4
+    cex: 6
+  # TRUE or FALSE, defaults to FALSE
+  add_box:
+  # "#04507d" for wet event & "#ca0020" for a dry event, defaults to wet
+  wheel_color: "#04507d"
+  # Integer, starting with 0 (use >= 1 to increase height)
+  wheel_hierarchy: 1
```
Comment (cnell-usgs) [inline callouts_cfg.yml:267]: Is this duplicate code in the script?

File: callouts_cfg.yml [hunk 2]
```diff
@@ -1,147 +1,207 @@
...
+  # Defaults to 9
+  fade_out:
+  text:
+    label: ["Hurricane Francine","brings high water","to the Gulf Coast"]
+    # This is in percentage space, relative to the plotting device
+    x_loc: 0.680
+    y_loc: 0.185
+    # Numeric indicating where to place text relative to location: 1,2,3,4 (below, left, above, right)
+    pos: 4
+    cex: 6
```
Comment (cnell-usgs) [inline callouts_cfg.yml:201]: This appears to be duplicate code added

**Comment by elmeraa-usgs** [comment]
Hmm.. Im not seeing any duplicate code. Those appear to be separate callouts with their associated dates, locations, etc.

**Review by cnell-usgs — APPROVED** [review]
oh I see. I misread it.

**Pull request merged**